packages/keplr/src/chains.ts

```diff
@@ -26,7 +25,6 @@ export async function setupChainInfo(
     currencies: chain.currencies,
     feeCurrencies: chain.feeCurrencies,
```

Comment:
```suggestion
    feeCurrencies: {
      ...chain.feeCurrencies,
      gasPriceStep: DesmosGasPriceStep,
    },
```
The `gasPriceStep` has been moved inside the `feeCurrencies` field you can see the doc [here](https://docs.keplr.app/api/suggest-chain.html).